static/css/targets.less

```diff
@@ -0,0 +1,139 @@
+.targets {
+  display: flex;
+  align-items: stretch;
+  flex-flow: row wrap;
+  justify-content: center;
+  padding: 35px 0;
+
+  .target {
+    padding: 10px;
+    width: 350px;
+    min-height: 180px;
+    background-color: #FFFFFF;
```

Comment:
Not `white`?

Reply:
Personal preference - I never use the reserved words for colours. I don't know why...